src/config.py

```diff
@@ -236,6 +238,18 @@ def add_debug_exporter(self):
                 {"verbosity": "basic"},
             )
 
+    def add_tls_insecure_skip_verify(self):
+        """Update `tls::insecure_skip_verify` in the otelcol config with the charm's config per exporter.
+
+        This allows the charm admin to skip verifying the certificate. Since we use the root cert
+        store we do not fine-grain the certs per exporter.
+        """
+        # Just update the section not overwrite the whole TLS section
+        for exporter in self._config["exporters"]:
+            self._config["exporters"][exporter].setdefault("tls", {}).setdefault(
+                "insecure_skip_verify", self._tls_insecure_skip_verify
+            )
+
```

Comment:
Let's a test to `unit/test_config.py`?
- Effect on empty config
- Effect on config with no exporters
- Effect on config with an exporter without tls section
- Effect on config with an exporter with an existing tls section